src/Groups/group_characters.jl

```diff
@@ -3583,6 +3583,43 @@ function symplectic_components(characters::Vector{GAPGroupClassFunction}, n::Int
                          GapObj(characters; recursive = true), n)]
 end
 
+@doc raw"""
+    character_table_complex_reflection_group(m::Int, p::Int, n::Int)
+
+Return the character table of the complex reflection group that is given by
+the input parameters.
+
+Note that this character table does not store an underlying group.
+
+# Examples
+```jldoctest
+julia> tbl = character_table_complex_reflection_group(3, 1, 2);
+
+julia> identifier(tbl)
+"C3wrS2"
+
+julia> order(tbl) == 3^2 * factorial(2)
+true
+
+julia> class_parameters(tbl)
+9-element Vector{Tuple{Vector{Int64}, Vector{Int64}}}:
+ ([1, 1], [])
+ ([1], [1])
+ ([1], [])
+ ([], [1, 1])
+ ([], [1])
+ ([], [])
+ ([2], [])
+ ([], [2])
+ ([], [])
```

Comment:
Just a stupid question: is it correct that two of the parameters are identical?

Reply:
Good spot!
Something is wrong with the translation of parameters from GAP to Oscar.